edge_computing/installing_with_siteconfig_operator/cnf-understanding-siteconfig-operator.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-understanding-siteconfig-operator"]
+= Understanding the {sco}
+include::_attributes/common-attributes.adoc[]
+:context: siteconfig-operator
+
+toc::[]
+
+With the new {sco}, you can deploy clusters with all available installation methods.
+The {sco} uses the unified `SiteConfig` API, which decouples parameters that define a cluster from the manner in which the cluster is deployed.
+This separation removes certain limitations that are presented by the `SiteConfig` kustomize plugin in the current xref:../../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-deploying-far-edge-sites[GitOps ZTP flow], such as agent cluster installations and scalability constraints posed by ArgoCD.
+Thanks to the unified `SiteConfig` API, the {sco} offers the following improvements:
+
+Consistency:: Whether you deploy your cluster with the Assisted Installer, the Image-based Installer, or any other flows based on your custom templates, the API is consistent with all installation methods.
+Scalability:: If you are using {ztp-first}, you can synchronize deployments with hundreds of clusters from a single `ClusterInstance` CR.
+Flexibility:: You can define your own set of manifest templates.
+Control:: You can declaratively specify sequenced operations, such as node deletion, in the `ClusterInstance` CR.
```

Comment:
```suggestion
Isolation:: Separates cluster definition from the installation method. The ClusterInstance (the new unified SiteConfig operator CRD) captures the cluster definition, while the Cluster Templates capture the cluster architecture and installation methods.
Unification:: The SiteConfig operator unifies both git and non-git workflows. Users can apply the `ClusterInstance` CR directly on the hub cluster or synchronize them through a GitOps solution like ArgoCD.
Consistency:: Maintains a consistent API across all installation methods, whether using the Assisted Installer, Image-based Installer, or any other custom template-based approach.
Scalability:: Achieves better scalability per cluster than siteconfig-v1. siteconfig-v1, configured as an ArgoCD application, can manage a maximum of 300 `SiteConfig` CRs, whereas siteconfig-v2 supports greater scalability.
Flexibility:: Provides users with more power to deploy and install clusters using custom cluster templates.
Troubleshooting:: Offers insightful information regarding cluster deployment status and rendered manifests, significantly enhancing the troubleshooting experience.
```

Reply:
> Scalability:: Achieves better scalability per cluster than siteconfig-v1. siteconfig-v1, configured as an ArgoCD application, can manage a maximum of 300 `SiteConfig` CRs, whereas siteconfig-v2 supports greater scalability.

Should we add an exact number for v2? Saying "up to x clusters" to facilitate comparison and better illustrate the scale of this improvement? I feel like "supports greater scalability" is too vague